package.json

```diff
@@ -32,10 +33,12 @@
     "levelup": "^5.0.0"
   },
   "devDependencies": {
+    "@types/levelup": "^4.3.3",
```

Comment:
I read 

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/935ee01761cb960c61cd3029da9aaeb8810090ba/types/levelup/index.d.ts#L1

is it not an issue since we have levelup 5?

Reply:
Looks like most of the imported base types are coming from here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/935ee01761cb960c61cd3029da9aaeb8810090ba/types/abstract-leveldown/index.d.ts, which has been updated to the 5.0 release already. From the CHANGELOG of levelup (https://github.com/Level/levelup/blob/master/CHANGELOG.md) the “breaking” parts are not related to its public API anyway….